README.org

```diff
@@ -92,17 +92,18 @@ This is how the sheets ~require~ in this project:
         - =mixins=
     - =sites/*.styl=: The site-specific sheets, as well as =generic.styl=, which applies to all of them, and also builds as a separate sheet for non-specific sites.  =all-sites.styl= puts all of the site-specific sheets into one big CSS file, which some people may prefer over setting up custom CSS for each site in their browser.
 
-This way, the color values are loaded first, followed by the dark- or light-themed mappings of those color values to friendly names, after which those friendly names can be used in the sheets that actually style elements and pages.
+This way, the color value-name mappings are loaded first, after which those friendly names can be used in the files that actually style elements and pages.
 
 *** New Themes
 
-It is possible to use this framework to build a theme for the web on your own specifications! Here's a short guide on creating your own themes.
+It's easy to add your own themes:
 
-1. Create a new folder in =themes/= to match your new theme.
-2. Copy the existing =themes/solarized/colors.styl= and =themes/solarized/dark.styl= into your directory.
-3. Modify the listed colors to match the base colors of your new theme!
 
-Adding additional or finer colors is possible, but requires more work, as the colors will need to be defined for every theme.
+1. Copy an existing theme directory in the =themes/= directory, giving it a new name.
+2. Modify it as required.
+3. Run =make= in the project root directory to build the CSS files.
+
+Adding additional, more detailed colors is possible, but requires more work, as the colors will need to be defined for every available theme.
```

Comment:
Thanks for making the other changes.

> Adding additional, more detailed colors is possible, but requires more work, as the colors will need to be defined for every available theme.

Could you help me understand what this line means?  I think the "additional, more detailed colors" part is a bit ambiguous--and if I don't know what it means, I doubt random people will either, haha.   ;)  

I guess it's referring to the color names, or color name-value mappings, like `color-background`, `emphasized`, etc.  Maybe what we need is a small section that explains that color names are defined in the non-theme files, and color names are applied to elements in the site-specific files, and the color names are mapped to color values in the theme-specific files.

If you'd rather, I could try to write that part myself instead of nit-picking your changes, haha.  Let me know.  Thanks for your work on this.

Reply:
Yup, I was trying to explain that they would need to add additional mappings, I do agree this part is quite vauge :).

I'm fine with removing it entirely if you think that would be better, it might be simply inferred once people take a look at the css layout.

Feel free to push changes to this branch if you know exactly the wording you would like! Otherwise, I'm happy to (try to) come up with something more clear. :)

Reply:
> I'm fine with removing it entirely if you think that would be better, it might be simply inferred once people take a look at the css layout.

I think that might be best for now.  I'd rather people refer to the source than possibly confuse them with an unclear explanation.  IOW, better no explanation than a confusing one.  :)

So let's go ahead and do that.  We can always add one later if we figure out a way to explain it well.

Thanks!